src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercial.tsx

```diff
@@ -143,7 +143,14 @@ export class ArtworkSidebarCommercialContainer extends React.Component<
     this.setState({ isErrorModalOpen: false })
   }
 
-  handleInquiry = () => {
+  @track((props: ArtworkSidebarCommercialContainerProps) => ({
```

Comment:
Used the props param here (but still needed to type it myself).

Reply:
```js
@track<ArtworkSidebarCommercialContainerProps>(props => { ... })
```

Reply:
(if you need state, the second arg to the generic takes state

Reply:
Oh nice, will update to the templated version.